### DISCUSSION TITLE
I can only access it from the LAN.

### DISCUSSION THREAD

**Discussion by lazyeo** [IceWhaleTech/CasaOS#1409] [Q&A]
I have no problems accessing CasaOS and the various Docker services installed in it from my LAN!

But when I try to access it from the outside via IP+port it is not accessible

I also can access it fine on my LAN using public IP + port, or I can use cloudflare tunnel for intranet penetration while I'm out there

Is there some default configuration that denies access to IPs from outside sources?

**Comment by thedroidkid** [comment]
Your router by default does not forward any ports outside of your network and that is a good thing. 

Do some research on using Cloudflare tunnels. It is not recommended to open any ports on your router.
